packages/analytics/analytics-chart/src/components/TopNTable.vue

```diff
@@ -6,12 +6,12 @@
       v-if="title"
       #title
     >
-      <div
+      <h2
```

Comment:
nit: you'll likely want the Analytics team to review this to ensure no surprises for Christian or Helen

Reply:
Pushed a commit to simplify these change a bit, but it shouldn't change visually/functionally, just the element. Will tag data team to make sure they are aware of this changes

Reply:
Oh looks like Mihai already saw it

Reply:
lol - yea just one quick heads up. If we went the `h2` route, it would turn into `var(--kui-font-size-50, 18px)`

And what we have now across the board for all Gateway Manager overview onboarding card Titles is `16px`. Would `<h3>` give us 16px font size?

Generally speaking, I'm all about using using heading tags to control font sizes, we'd just have to make sure all the other `.section-title`s are changed to match.

Reply:
Oh shoot you're right, KCard title is `18px` by default. that explains why we're overriding it so much everywhere. I'll ask design team whether we're fine making KCard title `16px` since it's the component we use for widgets by default.

Reply:
And no, we shouldn't use `h3` here because we're trying to standardize that widgets use `h2` as they are usually second level importance headings on the page after the page heading component (in most of our pages at least).

Reply:
I see...

I take back my comment about "using heading tags to control font sizes" – since each chart type (including TopNTable) could end up in an embedded dashboard, we probably should avoid using explicit heading tags inside the dashboard because it is not aware of the consuming app's heading hierarchy. 

Even though `<h2>` for the titles may work 90% of the time, I bet that eventually we would run into an heading structure issue.

Reply:
We could add `titleTag` prop here like we did in [EntityBaseTable.vue](https://github.com/Kong/public-ui-components/pull/1252/files#diff-fc2212f00918166c6240f9e29994448b13f8e9e154d57c506943a0467fcb596b). Would that work?

Reply:
We decided we'll add `titleTag` prop whenever possible to give a bit more flexibility in those components